src/jmclient/blockchaininterface.py

```diff
@@ -558,13 +563,20 @@ def query_utxo_set(self,
                 result.append(result_dict)
         return result
 
+    def _getmempoolinfo(self) -> Optional[dict]:
+        return self._rpc('getmempoolinfo')
+
     def _get_mempool_min_fee(self) -> Optional[int]:
-        rpc_result = self._rpc('getmempoolinfo')
+        rpc_result = self._getmempoolinfo()
         if not rpc_result:
             # in case of connection error:
             return None
         return btc.btc_to_sat(rpc_result['mempoolminfee'])
```

Comment:
Can we sanity check if `mempoolminfee` key is present in the dict? Or do we know that it always will be for the versions of Bitcoin Core we support?